test_scraping.py

```
@@ -0,0 +1,141 @@
+        for selector in price_selectors:
+            try:
+                elements = driver.find_elements(By.CSS_SELECTOR, selector)
+                print(f"  Found {len(elements)} elements with selector: {selector}")
+                
+                for element in elements[:5]:  # Check first 5 elements
+                    try:
+                        text = element.text.strip()
+                        if text and ('$' in text or text.isdigit()):
+                            prices_found.append(text)
+                            print(f"    Price text: {text}")
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)